src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="http://localhost:3000/api"
+      method="post"
+      class="searchform"
+      data-qa="big"
+    >
+      <label class="searchbar">
+        <img
+          src="images/Search.svg"
+          alt="Search_icon"
+          class="icon icon-big"
+        >
+        <input
+          type="search"
+          name="Search"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+          class="input input-big"
+          >
```

Comment:
closing brace should be at the same level as opening one - this way your layout is better structured and it is easier to distinguish where ends one tag and the other starts
```suggestion
        >
```

Reply:
Fixed.